packages/components/pie-button/package.json

```diff
@@ -8,6 +8,8 @@
   "types": "dist/types/index.d.ts",
   "scripts": {
     "build": "yarn build:wrapper pie-button && run -T vite build ",
+    "lint:scripts": "run -T eslint .",
+    "lint:scripts:fix": "run -T eslint . --fix",
```

Comment:
can you also please add these to the component generator if the update is across all components?

Reply:
I'll do that in a separate PR if you don't mind - I'll raise it after this one :)

Reply:
Quick note – this could just be `yarn lint:scripts --fix` (for that follow up PR)